modulefiles/RDAS/orion.intel.lua

```diff
@@ -83,7 +83,7 @@ setenv("RDASAPP_TESTDATA","/work2/noaa/da/cmartin/CI/RDASApp/data")
 setenv("RDAS_RRFS_DATA_ROOT","/work/noaa/fv3-cam/tlei/dr-emc-rdas/")
 prepend_path("PATH","/apps/contrib/NCEP/libs/hpc-stack/intel-2018.4/prod_util/1.2.2/bin")
 
-execute{cmd="ulimit -s unlimited",modeA={"load"}}
+--cltorg execute{cmd="ulimit -s unlimited",modeA={"load"}}
```

Comment:
is this correct to remove this?

Reply:
Do others have this issue? It sounds like something specific to your environment

Reply:
```cat ~tlei/.bashrc
...
ulimit -s 1900000
...
```
You set the stack size at login, it probably doesn't let you re-set it

Reply:
@CoryMartin-NOAA  To make sure if that line in the  .bashrc does have any impact, I commented out it and re-ran the building and it was the same behavior.  But interestingly, if I directly ran "module load " on the front end, there is no problem.

Reply:
@TingLei-NOAA this is very strange

Reply:
If no one else has the same problem I will do some digging when I have a chance :).

Reply:
I was able to successfully run the mpasjedi case on Orion by sourcing the default orion module file (i.e., keeping the unlimited stack size command). This was done using a basic job file and not through the ctest, though.

Reply:
@SamuelDegelia-NOAA  so your running of the build.sh (which would use that module ) has no problem?

Reply:
@TingLei-NOAA Correct, I had no problem building using the default build.sh. I just followed the wiki instructions line for line and everything worked for me.

Reply:
@SamuelDegelia-NOAA  Thanks. then seems some set on my side caused that issue.

Reply:
Is there a specific error message that occurs when you build?

Reply:
The error message said those re-set are not allowed.